### ISSUE TITLE
SSL certificates are outdated even on a fresh installation.

### ISSUE THREAD

**Issue by ljlismybaby** [yt-dlp/yt-dlp#8376]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm requesting a feature unrelated to a specific site
- [X] I've looked through the [README](https://github.com/yt-dlp/yt-dlp#readme)
- [X] I've verified that I'm running yt-dlp version **2023.10.13** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)

### Provide a description that is worded well enough to be understood

When i download the yt_tub in the website ,my compter is the macOS, I want to download a veido,she tell me my SLL don't have the right power.(no workarounds)
![image](https://github.com/yt-dlp/yt-dlp/assets/148359060/0dca5044-cfaf-476e-9a9d-a49abe479ecf)
Updating yt-dlp
throws an error: (macOS
![image](https://github.com/yt-dlp/yt-dlp/assets/148359060/e406e838-e3b3-4c69-b224-51d0a03949cf)

### Provide verbose output that clearly demonstrates the problem

- [ ] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [ ] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [ ] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
*[youtube] jNQXAC9IVRw: Unable to download API page: [SSL: CERTIFICATE_VER IFY_FAILED] certificate verify failed: unable to get local issuer certificate (. ss1.c:1000) (caused by CertificateVerifyError ('[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ss1.c: 1000) )); please report this issue on https: //github.com/yt-dIp/yt-dip/issues?q= , fi lling out the appropriate issue template. Confirm you are on the latest version using yt-d1p-U
```

**Comment by bashonly** [comment]
Can you post the **full** output of `yt-dlp -v`?

How did you install yt-dlp?

**Comment by october262** [comment]
try with yt-dlp --no-check-certificate " jNQXAC9IVRw"

**Comment by bashonly** [comment]
@october262 that is not a long-term solution

**Comment by ljlismybaby** [comment]
OK，I will try it

发送自 Outlook for iOS<https://aka.ms/o0ukef>
________________________________
发件人: bashonly ***@***.***>
发送时间: Wednesday, October 18, 2023 8:38:55 PM
收件人: yt-dlp/yt-dlp ***@***.***>
抄送: ljlismybaby ***@***.***>; Author ***@***.***>
主题: Re: [yt-dlp/yt-dlp] SSL certificates are outdated even on a fresh installation. (Issue #8376)


@october262<https://github.com/october262> that is not a long-term solution

―

**Comment by ljlismybaby** [comment]
in a mac terminal window, i used the sudo curl and chmod commands (the first option listed in the wiki), and i’ll send you the output in just a second

发送自 Outlook for iOS<https://aka.ms/o0ukef>
________________________________
发件人: yifan du ***@***.***>
发送时间: Wednesday, October 18, 2023 8:57:43 PM
收件人: yt-dlp/yt-dlp ***@***.***>; yt-dlp/yt-dlp ***@***.***>
抄送: Author ***@***.***>
主题: Re: [yt-dlp/yt-dlp] SSL certificates are outdated even on a fresh installation. (Issue #8376)

OK，I will try it

发送自 Outlook for iOS<https://aka.ms/o0ukef>
________________________________
发件人: bashonly ***@***.***>
发送时间: Wednesday, October 18, 2023 8:38:55 PM
收件人: yt-dlp/yt-dlp ***@***.***>
抄送: ljlismybaby ***@***.***>; Author ***@***.***>
主题: Re: [yt-dlp/yt-dlp] SSL certificates are outdated even on a fresh installation. (Issue #8376)


@october262<https://github.com/october262> that is not a long-term solution

―

**Comment by bashonly** [comment]
Try re-installing yt-dlp with these commands:
```
sudo curl -L https://github.com/yt-dlp/yt-dlp/releases/latest/download/yt-dlp_macos_legacy -o /usr/local/bin/yt-dlp
sudo chmod a+rx /usr/local/bin/yt-dlp
```

**Comment by gamer191** [comment]
> i’ll send you the output in just a second

Please send the output of yt-dlp -v

**Comment by coletdjnz** [comment]
You likely will need to install or update certifi, or fix your system Python certificates.

**Comment by AndrewUshakov** [comment]
Hi!
Just got the same error:
`
(.venv312) C:\Users\UshakovAY\PycharmProjects\yt_dlp>yt-dlp -vU H0mrAJB4rrs                                                             
[debug] Command-line config: ['-vU', 'H0mrAJB4rrs']
[debug] Encodings: locale cp1251, fs utf-8, pref cp1251, out utf-8, error utf-8, screen utf-8 
[debug] yt-dlp version stable@2023.10.13 [b634ba742] (pip) 
[debug] Python 3.12.0 (CPython AMD64 64bit) - Windows-10-10.0.19044-SP0 (OpenSSL 3.0.11 19 Sep 2023) 
[debug] exe versions: none 
[debug] Optional libraries: Cryptodome-3.19.0, brotli-1.1.0, certifi-2023.07.22, mutagen-1.47.0, sqlite3-3.42.0, websockets-12.0
[debug] Proxy map: {}
[debug] Loaded 1890 extractors 
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest 
Available version: stable@2023.10.13, Current version: stable@2023.10.13 
yt-dlp is up to date (stable@2023.10.13) 
[youtube] Extracting URL: H0mrAJB4rrs 
[youtube] H0mrAJB4rrs: Downloading webpage 
WARNING: [youtube] Unable to download webpage: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1000) 
[youtube] H0mrAJB4rrs: Downloading ios player API JSON 
WARNING: [youtube] [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1000). Retrying (1/3)... 
[youtube] H0mrAJB4rrs: Downloading ios player API JSON 
WARNING: [youtube] [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1000). Retrying (2/3)... 
[youtube] H0mrAJB4rrs: Downloading ios player API JSON 
WARNING: [youtube] [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1000). Retrying (3/3)... 
[youtube] H0mrAJB4rrs: Downloading ios player API JSON 
[youtube] H0mrAJB4rrs: Downloading android player API JSON 
WARNING: [youtube] [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1000). Retrying (1/3)... 
[youtube] H0mrAJB4rrs: Downloading android player API JSON 
WARNING: [youtube] [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1000). Retrying (2/3)... 
[youtube] H0mrAJB4rrs: Downloading android player API JSON 
WARNING: [youtube] [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1000). Retrying (3/3)... 
[youtube] H0mrAJB4rrs: Downloading android player API JSON 
WARNING: [youtube] Unable to download API page: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1000) (caused by CertificateVerifyError('[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ss
l.c:1000)')); please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
[youtube] H0mrAJB4rrs: Downloading iframe API JS 
WARNING: [youtube] Unable to download webpage: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1000) 
[youtube] H0mrAJB4rrs: Downloading web player API JSON 
WARNING: [youtube] [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1000). Retrying (1/3)... 
[youtube] H0mrAJB4rrs: Downloading web player API JSON 
WARNING: [youtube] [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1000). Retrying (2/3)... 
[youtube] H0mrAJB4rrs: Downloading web player API JSON 
WARNING: [youtube] [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1000). Retrying (3/3)... 
[youtube] H0mrAJB4rrs: Downloading web player API JSON 
WARNING: [youtube] Unable to download API page: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1000) (caused by CertificateVerifyError('[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ss
l.c:1000)')); please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
ERROR: [youtube] H0mrAJB4rrs: Unable to download API page: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1000) (caused by CertificateVerifyError('[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certi
ficate (_ssl.c:1000)')); please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
  File "C:\Users\UshakovAY\PycharmProjects\yt_dlp\.venv312\Lib\site-packages\yt_dlp\extractor\common.py", line 715, in extract
    ie_result = self._real_extract(url)
                ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\UshakovAY\PycharmProjects\yt_dlp\.venv312\Lib\site-packages\yt_dlp\extractor\youtube.py", line 4073, in _real_extract
    webpage, master_ytcfg, player_responses, player_url = self._download_player_responses(url, smuggled_data, video_id, webpage_url)
                                                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\UshakovAY\PycharmProjects\yt_dlp\.venv312\Lib\site-packages\yt_dlp\extractor\youtube.py", line 4037, in _download_player_responses
    player_responses, player_url = self._extract_player_responses(
                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\UshakovAY\PycharmProjects\yt_dlp\.venv312\Lib\site-packages\yt_dlp\extractor\youtube.py", line 3728, in _extract_player_responses
    raise last_error
  File "C:\Users\UshakovAY\PycharmProjects\yt_dlp\.venv312\Lib\site-packages\yt_dlp\extractor\youtube.py", line 3693, in _extract_player_responses
    pr = initial_pr if client == 'web' and initial_pr else self._extract_player_response(
                                                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\UshakovAY\PycharmProjects\yt_dlp\.venv312\Lib\site-packages\yt_dlp\extractor\youtube.py", line 3613, in _extract_player_response
    return self._extract_response(
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\UshakovAY\PycharmProjects\yt_dlp\.venv312\Lib\site-packages\yt_dlp\extractor\youtube.py", line 966, in _extract_response
    next(main_retries)
  File "C:\Users\UshakovAY\PycharmProjects\yt_dlp\.venv312\Lib\site-packages\yt_dlp\utils\_utils.py", line 5114, in __iter__
    self.error_callback(self.error, self.attempt, self.retries)
  File "C:\Users\UshakovAY\PycharmProjects\yt_dlp\.venv312\Lib\site-packages\yt_dlp\extractor\common.py", line 3760, in _error_or_warning
    RetryManager.report_retry(
  File "C:\Users\UshakovAY\PycharmProjects\yt_dlp\.venv312\Lib\site-packages\yt_dlp\utils\_utils.py", line 5122, in report_retry
    raise e
  File "C:\Users\UshakovAY\PycharmProjects\yt_dlp\.venv312\Lib\site-packages\yt_dlp\extractor\youtube.py", line 955, in _extract_response
    response = self._call_api(
               ^^^^^^^^^^^^^^^
  File "C:\Users\UshakovAY\PycharmProjects\yt_dlp\.venv312\Lib\site-packages\yt_dlp\extractor\youtube.py", line 603, in _call_api
    return self._download_json(
           ^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\UshakovAY\PycharmProjects\yt_dlp\.venv312\Lib\site-packages\yt_dlp\extractor\common.py", line 1069, in download_content
    res = getattr(self, download_handle.__name__)(url_or_request, video_id, **kwargs)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\UshakovAY\PycharmProjects\yt_dlp\.venv312\Lib\site-packages\yt_dlp\extractor\common.py", line 1033, in download_handle
    res = self._download_webpage_handle(
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\UshakovAY\PycharmProjects\yt_dlp\.venv312\Lib\site-packages\yt_dlp\extractor\common.py", line 903, in _download_webpage_handle
    urlh = self._request_webpage(url_or_request, video_id, note, errnote, fatal, data=data, headers=headers, query=query, expected_status=expected_status)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\UshakovAY\PycharmProjects\yt_dlp\.venv312\Lib\site-packages\yt_dlp\extractor\common.py", line 860, in _request_webpage
    raise ExtractorError(errmsg, cause=err)

  File "C:\Program Files\Python312\Lib\urllib\request.py", line 1344, in do_open
    h.request(req.get_method(), req.selector, req.data, headers,
  File "C:\Program Files\Python312\Lib\http\client.py", line 1319, in request
    self._send_request(method, url, body, headers, encode_chunked)
  File "C:\Program Files\Python312\Lib\http\client.py", line 1365, in _send_request
    self.endheaders(body, encode_chunked=encode_chunked)
  File "C:\Program Files\Python312\Lib\http\client.py", line 1314, in endheaders
    self._send_output(message_body, encode_chunked=encode_chunked)
  File "C:\Program Files\Python312\Lib\http\client.py", line 1074, in _send_output
    self.send(msg)
  File "C:\Program Files\Python312\Lib\http\client.py", line 1018, in send
    self.connect()
  File "C:\Program Files\Python312\Lib\http\client.py", line 1460, in connect
    self.sock = self._context.wrap_socket(self.sock,
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Program Files\Python312\Lib\ssl.py", line 455, in wrap_socket
    return self.sslsocket_class._create(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Program Files\Python312\Lib\ssl.py", line 1046, in _create
    self.do_handshake()
  File "C:\Program Files\Python312\Lib\ssl.py", line 1317, in do_handshake
    self._sslobj.do_handshake()
ssl.SSLCertVerificationError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1000)

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\UshakovAY\PycharmProjects\yt_dlp\.venv312\Lib\site-packages\yt_dlp\extractor\common.py", line 847, in _request_webpage
    return self._downloader.urlopen(self._create_request(url_or_request, data, headers, query))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\UshakovAY\PycharmProjects\yt_dlp\.venv312\Lib\site-packages\yt_dlp\YoutubeDL.py", line 4051, in urlopen
    return self._request_director.send(req)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\UshakovAY\PycharmProjects\yt_dlp\.venv312\Lib\site-packages\yt_dlp\networking\common.py", line 114, in send
    response = handler.send(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\UshakovAY\PycharmProjects\yt_dlp\.venv312\Lib\site-packages\yt_dlp\networking\_helper.py", line 204, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\UshakovAY\PycharmProjects\yt_dlp\.venv312\Lib\site-packages\yt_dlp\networking\common.py", line 325, in send
    return self._send(request)
           ^^^^^^^^^^^^^^^^^^^
  File "C:\Users\UshakovAY\PycharmProjects\yt_dlp\.venv312\Lib\site-packages\yt_dlp\networking\_urllib.py", line 424, in _send
    handle_response_read_exceptions(cause)
  File "C:\Users\UshakovAY\PycharmProjects\yt_dlp\.venv312\Lib\site-packages\yt_dlp\networking\_urllib.py", line 342, in handle_response_read_exceptions
    handle_sslerror(e)
  File "C:\Users\UshakovAY\PycharmProjects\yt_dlp\.venv312\Lib\site-packages\yt_dlp\networking\_urllib.py", line 334, in handle_sslerror
    raise CertificateVerifyError(cause=e) from e
yt_dlp.networking.exceptions.CertificateVerifyError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1000)
`

**Comment by RingoTheDog** [comment]
to update Python security certs:

pip install --upgrade certifi
